code/a/validate_con_imputation.do

```diff
@@ -0,0 +1,148 @@
+/******************************************************************************************/
+/* test accuracy of imputation process by dropping additional shrids and seeing how we do */
+/******************************************************************************************/
+/* design decisions / notes:
+- do 2008 con boundaries only
+- work with 2001, where we need almost no imputation, so our reference test is damn good.
+- values used in SHRUG are in $shrug/intermediate/con_pre_impute_08.
+- goal is to compare what we generate to these ones
+
+1. set X% of 2001 populations to missing
+2. rerun the imputation to get constituency population totals
+3. compare the totals to $shrug/intermediate/con_pre_impute_08.
+- mean absolute deviation as a percentage of constituency population
+
+4. graph: share shrids dropped on X axis, mean absolute deviation on Y axis.
+
+*/
+
+/**********************************************************************************************************************/
+/* program impute_shrid_pop : Impute population for missing villages based on other villages in the same constituency */
+/*   note: this is copy-pasted from collapse_shrug_to_con.do exactly */
```

Comment:
replicators don't know what `collapse_shrug_to_con.do` is since it's not in the replication repo. delete this line

Reply:
fixed in this [commit](https://github.com/devdatalab/paper-shrug/commit/19a059b092ad2fa54f49dccc7f3808daae340eb1)